process-compose/README.md

```diff
@@ -0,0 +1,41 @@
+# What is this?
```

Comment:
Did you source the schema from somewhere external or fill it out yourself?

It might be worth leaving instructions on how we can update the `schema.yaml` in future if there are repeatable steps (or even if you manually wrote it up)